### ISSUE TITLE
Set Modeldoc's default culling type to "none"

### ISSUE THREAD

**Issue by Grodbert** [Facepunch/sbox-issues#3213]
### How would you like it to work?

Usually, when bones from your model are getting culled, you have to add a bonemarkup node to say not to discard bones, or go to the markup section and change the culling type from "aggressive" to "none".

I believe this system has never helped anyone and is only a cause of conchfusion for newcomers and a source of annoyance for everyone else, it should be set to "none" by default.

Other? Comment.

**Comment by garrynewman** [comment]
I agree with this

**Comment by Gmod4phun** [comment]
Agree, if the bones are in the model, they probably need to be there in 99% of cases even if they don't have a mesh skinned to them.

**Comment by aylaylay** [comment]
All bones are now kept

**Comment by Grodbert** [comment]
I can confirm that it now keeps all unskinned bones, despite bonemarkuplist still being set by default to "aggressive"

![image](https://github.com/sboxgame/issues/assets/86626409/6dea07a9-b2ba-46e2-9f26-d741207b0767)

It might be worth to delete the little section altogether, although I could see how it could be of some use to have it set to "none" by default instead.
I'll leave that up to the developers, but in the meantime, it works for me, so I'll mark as completed.

**Comment by aylaylay** [comment]
Yeah I'm considering removing the markup shit, the agressive thing I could only see being used in animation compiling though

**Comment by Grodbert** [comment]
![Image](https://github.com/user-attachments/assets/a63185b8-f485-41f5-ad4f-75e9272843e7)

not sure when it happened, but bone culling is set to aggressive by default again
